### ISSUE TITLE
Move CorrelationId from NLogLogger class to CSOMOperation class

### ISSUE THREAD

**Issue by OndrejValenta** [KeenMate/fluently-sharepoint#7]
So it can be set on an existing CSOMOperation through an extension method at any time. 

Something like this:
````
var op = "https://keenmate.sharepoint.com/sites/demo/manufacturingfacility"
		.Create(new NlogLogger())
		.SetOnlineCredentials(username, password)
		.SetCorrelationId(Guid.NewGuid())
		.LoadList("Stock Items")
		.Execute()
		.SetCorrelationId(Guid.NewGuid()) // new operation group
		.CreateList(....)
		.Execute();

**Comment by albertmoravec** [comment]
Resolved by 63f57818eadeb25c8cb438a4777250bf28c6b1a6

**Comment by albertmoravec** [comment]
357bf3d70c34a8d1ef7840f2b0c21bdbca5dc33d introduces a few WebTemplate constants and 609af579076d0fd54b52b9f0ec013d83984c51c9 added enum containing LCID keys for common languages
